anchor/common/ssv_types/src/cluster.rs

```diff
@@ -0,0 +1,53 @@
+use crate::OperatorId;
+use crate::Share;
+use derive_more::{Deref, From};
+use types::{Address, Graffiti, PublicKey};
+
+/// Unique identifier for a cluster
+#[derive(Clone, Copy, Debug, Default, Eq, PartialEq, Hash, From, Deref)]
+pub struct ClusterId(pub u64);
+
+/// A Cluster is a group of Operators that are acting on behalf of a Validator
+#[derive(Debug, Clone)]
+pub struct Cluster {
```

Comment:
I think domain needs to be included on the Cluster unless you had another plan for that implementation

Reply:
I left domain and some other specific Validator data off since a lot of it is going to be included via the lighthouse types. If we realize it makes more sense to add it here we can do that.